app/services/container_topology_service.rb

```diff
@@ -114,15 +114,12 @@ def retrieve_providers
   end
 
   def build_kinds
-    kinds =  { :Replicator => true,
-               :Pod        => true,
-               :Container  => true,
-               :Node       => true,
-               :Service    => true,
-               :Host       => true,
-               :VM         => true,
-               :Route      => true
-             }
+    base_kinds = [:Replicator, :Pod, :Container, :Node, :Service, :Host, :VM, :Route]
+    kinds = {}
+    base_kinds.each do |kind|
+      kinds[kind] = true
+    end
```

Comment:
I was curious if there was another way to do it and came up with...

```ruby
base_kinds = %i(Replicator Pod Container Node Service Host VM Route)
base_kinds.zip([true] * base_kinds.size).to_h
```

Not necessary to change...was just for fun :)  cc @matthewd